### ISSUE TITLE
Not showing console.readLine(..) messagens

### ISSUE THREAD

**Issue by fbrubbo** [scd4j/gradle-plugins#25]
Gradle shows **> Building 0% > :scd4j** whenever console.readLine(..) is used.

Try to execute the CreateUserAndChangeFileOwnership

```
====== Version Info ===================
> Building 0% > :scd4j
Review the above config. Type "yes/y" to procceed or anSCD4J Version : 0.7.20
Gradle Version: 2.14
Jvm Version   : 1.8.0_91 (Oracle Corporation 25.91-b14)
Pack Name     : CreateUserAndChangeFileOwnership 
Pack Version  : 0.1 
====== Environment Configuration ======
IP PRODUCTION LIST      : []
IP STAGING LIST         : []
IP TESTING LIST         : []
IP DESENV  LIST         : [ANY OTHER]
====== Instalation Configuration ======
CONFIG FILE   : /home/fernando/git/gradle-plugins/scd4j-examples/CreateUserAndChangeFileOwnership/config/example.conf
MODULE DIRS   : [/home/fernando/git/gradle-plugins/scd4j-examples/CreateUserAndChangeFileOwnership/modules/example-module1]
==================================================================
> Building 0% > :scd4j

```

**Comment by fbrubbo** [comment]
This is a [bug in gradle 2.13](https://issues.gradle.org/browse/GRADLE-3446). We are going to rollback it to 2.12 up until the fix is ready.

**Comment by fbrubbo** [comment]
We have closed this issue, but a new one was created(#26) for further gradle update
